src/components/NewsletterSignup/NewsletterSignup.stories.tsx

```diff
@@ -320,6 +346,45 @@ export const ComponentStates: Story = {
   ),
 };
 
+export const HightlightColors: Story = {
+  render: () => (
+    <VStack align="stretch" spacing="l">
+      <Box>
+        <Heading level="h3" size="heading6">
+          Education
+        </Heading>
+        <NewsletterSignup
+          id="highlight-color-education"
+          view="form"
+          onChange={() => {}}
+          onSubmit={() => {}}
+          title="Education Newsletter"
+          highlightColor="section.education.primary"
+          confirmationHeading={confirmationHeading}
+          errorHeading={errorHeading}
+          descriptionText={descriptionText}
+        />
+      </Box>
+      <Box>
+        <Heading level="h3" size="heading6">
+          Research Library Schomburg
+        </Heading>
+        <NewsletterSignup
+          id="highlight-color-schomburg"
+          view="form"
+          onChange={() => {}}
+          onSubmit={() => {}}
+          title="Schomburg Newsletter"
```

Comment:
Change to `Schomburg Newsletter Color`.

Reply:
I removed the second example because with the code snipped added I thought it makes more sense just to have one that first shows the code and then the rendered component.

Reply:
Ok, makes sense.